pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -96,12 +87,7 @@ public HelixBrokerStarter(String helixClusterName, String zkServer, Configuratio
 
     _pinotHelixProperties.addProperty("pinot.broker.id", brokerId);
     setupHelixSystemProperties();
-    RoutingTableBuilder defaultOfflineRoutingTableBuilder =
-        getRoutingTableBuilder(_pinotHelixProperties.subset(DEFAULT_OFFLINE_ROUTING_TABLE_BUILDER_KEY));
-    RoutingTableBuilder defaultRealtimeRoutingTableBuilder =
-        getRoutingTableBuilder(_pinotHelixProperties.subset(DEFAULT_REALTIME_ROUTING_TABLE_BUILDER_KEY));
-    Map<String, RoutingTableBuilder> tableToRoutingTableBuilderMap =
-        getTableToRoutingTableBuilderMap(_pinotHelixProperties.subset(ROUTING_TABLE_BUILDER_KEY));
+    // TODO read config for ROUTING_TABLE_SELECTOR_CLASS_KEY, construct (or load) a RoutingTableSelector class appropriately
```

Comment:
Is this still necessary considering the fact that there is the routing table selector factory that switches between routing table selector implementations?